testing/mod.ts

```diff
@@ -69,8 +69,19 @@ function print(txt: string, newline: boolean = true): void {
   Deno.stdout.writeSync(encoder.encode(`${txt}`));
 }
 
+declare global {
+  interface Window {
+    __DENO_TEST_REGISTRY: TestDefinition[];
```

Comment:
How about sticking it on the Deno object - like Deno.testRegistry ?

Reply:
Yes, that's fine for me :+1: thanks

Reply:
Probably there needs to be an explanatory comment about it's purpose - I think it will be confusing to readers

Reply:
@ry I think it won't be possible - firstly I would need to overwrite `Deno` namespace and additionally `window.Deno` is frozen - we can't add any new properties to it